src/lib/components/Timeline/TimelineItem.tsx

```diff
@@ -0,0 +1,18 @@
+import { FC, PropsWithChildren, ComponentProps } from 'react';
+import classNames from 'classnames';
+import { useTimelineContext } from './TimelineContext';
+
+export type TimelineItemProps = PropsWithChildren<ComponentProps<'li'>>;
+
+export const TimelineItem: FC<TimelineItemProps> = ({ children, ...props }) => {
+  const { horizontal } = useTimelineContext();
+  return (
+    <li
+      data-testid="timeline-item"
+      className={classNames({ 'mb-10 ml-6': !horizontal, 'relative mb-6 sm:mb-0': horizontal })}
+      {...props}
```

Comment:
For all components that we spread the rest of the props that the user can pass we need to extract the className prop and append it to the one we have otherwise it will get overridden.

Reply:
Oh I see what you mean because `...props` contains `className` as well. I'll fix it.